### PR TITLE
feat: migrated leaflet map to LWC

### PR DESCRIPTION
**Pull request by pozil**
Migrated the leaflet map in Property Finder from Aura to LWC

### CODE REVIEW

File: force-app/main/default/lwc/propertyListMap/__tests__/propertyListMap.test.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        document.body.appendChild(element);
+
+        // Mock handler for toast event
+        const handler = jest.fn();
+        // Add event listener to catch toast event
+        element.addEventListener(ShowToastEventName, handler);
+
+        // Wait for any asynchronous DOM updates
+        await flushPromises();
+        await flushPromises();
```
Comment (albarivas) [inline force-app/main/default/lwc/propertyListMap/__tests__/propertyListMap.test.js:103]: This is done twice!

**Review by albarivas — CHANGES_REQUESTED** [review]
I left a few minor comments, please take a look!

File: force-app/main/default/lwc/propertyListMap/__tests__/propertyListMap.test.js (hunk 1 above)
Comment (pozil) [reply force-app/main/default/lwc/propertyListMap/__tests__/propertyListMap.test.js:103]: Yes, it's needed twice. I'll add some comments on it.

Comment (pozil) [reply force-app/main/default/lwc/propertyListMap/__tests__/propertyListMap.test.js:104]: Added some comments to explain why we need to wait twice here.

File: package.json
[the remark this answers is not in the document]
Comment (pozil) [reply package.json:1]: Done.

**Review by albarivas — APPROVED** [review]

**Pull request merged**